src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -566,7 +581,9 @@ public Task SendFunctionEnvironmentReloadRequest()
                 FunctionEnvironmentReloadRequest = request
             });
 
-            return _reloadTask.Task;
+            var environmentReloadRequestResult = _reloadTask.Task;
+
+            return environmentReloadRequestResult;
```

Comment:
is this change needed?

Reply:
not needed. Revered this in the latest push,.